CONTRIBUTORS.txt

```diff
@@ -95,6 +95,7 @@ Enrico Abcede            (@emabcede30)          September 2024
 Ilaria Oliveti           (@IlariaOliveti)       October 2024
 Catarina Costa           (@catarinaquintela)    February 2025
 Mara Mita                (@MaraMita)            February 2025
+Valeria Cascone          (@ValeriaCascone)      February 2025
```

Comment:
We are in March ;-)

Reply:
HHAHAH... yes, but in spirit, I feel we are still in February!!!